rate/rate.py

```diff
@@ -0,0 +1,228 @@
+# coding: utf-8
+"""
+This is an XBlock designed to allow people to provide feedback on our
+course resources.
+"""
+
+import random
+
+import pkg_resources
+
+from xblock.core import XBlock
+from xblock.fields import Scope, Integer, String, List, Float
+from xblock.fragment import Fragment
+
+try:
+    from eventtracking import tracker
+except ImportError:
+    class tracker(object):  # pylint: disable=invalid-name
+        """
+        Define tracker if eventtracking cannot be imported. This is a workaround
```

Comment:
Thanks for adding eventing. It's a pain to have to do this workaround in every xblock-- have you thought about pushing the workaround down into the xblock repo itself (or fixing emit)? I don't know if there is any other way for xblocks to share utility code like this.